docs/explanation/index.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Keeping the file in the repo for now instead of deleting it but using `:orphan:` directive to exclude it from TOC trees without triggering warnings.